content/en/docs/concepts/policy/limit-range.md

```diff
@@ -1,64 +1,64 @@
 ---
 reviewers:
 - nelvadas
-title: Limit Ranges
+title: LimitRanges
 content_template: templates/concept
 weight: 10
 ---
 
 {{% capture overview %}}
 
 By default, containers run with unbounded [compute resources](/docs/user-guide/compute-resources) on a Kubernetes cluster.
-With Resource quotas, cluster administrators can restrict the resource consumption and creation on a namespace basis.
-Within a namespace, a Pod or Container can consume as much CPU and memory as defined by the namespace's resource quota. There is a concern that one Pod or Container could monopolize all of the resources. Limit Range is a policy to constrain resource  by Pod or Container in a namespace.
+With resource quotas, cluster administrators can restrict resource consumption and creation on a namespace basis.
+Within a namespace, a Pod or Container can consume as much CPU and memory as defined by the namespace's resource quota. There is a concern that one Pod or Container could monopolize all available resources. A LimitRange is a policy to constrain resource consumption by Pod or Container in a namespace.
```

Comment:
How about: “A LimitRange is a policy to constrain resource allocations (to Pods or Containers) in a namespace.”

One thing I want readers to learn is that LimitRanger can enforce _minimum_ request (and _minimum_ limit). So if I want to create a Pod that requests 5m CPU and 16KiB memory, someone who manages that namespace can force me to request more CPU and more memory. I'm thus restricted in my ability to mislead the scheduler.